### PR TITLE
Overload Step#expectNext

### PR DESCRIPTION
**Pull request by jakekdodd**
I found myself having to use `@SuppressWarnings("unchecked")` on all of my test methods because of the generic varargs array creation. For most use cases (where people are checking 1-6 values), this change obviates the need for the annotation.

Ideally I would have made these default methods, but unfortunately that's not possible (as far as I can tell).

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@jakekdodd Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-addons&pullRequestId=106)!

[Click here](https://cla.pivotal.io/sync/reactor?repositoryId=reactor/reactor-addons&pullRequestId=106) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: reactor-test/src/main/java/reactor/test/StepVerifier.java [hunk 1]
```diff
@@ -513,6 +595,17 @@
...
 		/**
+		 * Expect the next elements received to be equal to the given values.
+		 *
+		 * @param ts the values to expect
+		 *
+		 * @return this builder
+		 *
+		 * @see Subscriber#onNext(Object)
+		 */
+		Step<T> expectNext(List<T> ts);
```
Comment (simonbasle) [inline reactor-test/src/main/java/reactor/test/StepVerifier.java]: isn't that already covered by `expectNextSequence`?

**Comment by simonbasle** [comment]
awesome, yeah that's certainly alleviating a small pain point (and expectNext is probably the most often used expectation method)

File: reactor-test/src/main/java/reactor/test/StepVerifier.java (hunk 1 above)
Comment (jakekdodd) [inline reactor-test/src/main/java/reactor/test/StepVerifier.java]: Ah yes it is, I'll remove this method 😄

**Comment by simonbasle** [comment]
Thanks @jakekdodd, I'll be happy to merge this PR once you sign the CLA (see @pivotal-issuemaster message above for links)

**Comment by jakekdodd** [comment]
@simonbasle just waiting for the lawyers to give me the go-ahead to sign the CLA 😄

**Comment by pivotal-issuemaster** [comment]
@jakekdodd Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-addons&pullRequestId=106)!

**Pull request merged**